src/ckb-daemon/structures.h

```diff
@@ -247,7 +248,7 @@ typedef struct {
     // Color dithering in use
     char dither;
     // Flag to check, if large macros should be sent delayed
-    char delay;
+    uint delay;
```

Comment:
This var is used by the GUI element "Settings -> More Settings -> Use delay for very long macros.
Have you checked what happens with this Element if you change the type?